pootle/settings/91-travis.conf

```diff
@@ -23,6 +23,9 @@ if os.environ.get("TRAVIS"):
             'HOST': '',
             'PORT': '',
             'ATOMIC_REQUESTS': True,
+            'TEST': {
+                'NAME': 'test_pootle',
+            }
```

Comment:
@dwaynebailey i think we need to update this in the default dbs dict, and probs in the `pootle init` command too